docs/.vuepress/config.js

```diff
@@ -32,13 +32,15 @@ module.exports = {
               {
                 text: 'Development (master)',
                 link: '/chartjs-plugin-annotation/master/',
+                target: '_self',
```

Comment:
What does this do?

Reply:
Opens in same tab. This does not change for old versions of docs though.